### DISCUSSION TITLE
Greenfield API - Get available (confirmed) und unavailable (unconfirmed) wallet balance

### DISCUSSION THREAD

**Discussion by g33kme** [btcpayserver/btcpayserver#2419] [Ideas / Feature Requests]
Some feature request for the greenfield API. It seems I get my wallet balance for all (confirmed, unconfirmed) transactions. Is there a way to see confirmed and unconfirmed balance for the wallet?

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver/pull/2420

**Reply by NicolasDorier** [reply]
merged, can close this discussion.

**Reply by pavlenex** [reply]
Discussions cannot be closed (yet). Perhaps GitHub will add this feature in the future.
